apps/test/unit/accounts/ChangeEmailFormTest.jsx

```diff
@@ -154,67 +153,6 @@ describe('ChangeEmailForm', () => {
     });
   });
 
-  describe('calls onSubmit', () => {
-    let onSubmit, wrapper;
-
-    beforeEach(() => {
-      onSubmit = sinon.spy();
-      wrapper = mount(
-        <ChangeEmailForm {...DEFAULT_PROPS} onSubmit={onSubmit} />
-      );
-    });
-
-    it('when the enter key is pressed in the email field', () => {
```

Comment:
Do we want to keep the 'Enter'-pressing behavior?

Reply:
since the `TextField` design system component doesn't expose the `onKeyDown` event listener as a prop yet, I decided to nix the behavior. it didn't seem 100% necessary in the moment. happy to bring it back if it's a priority. to add it back, the best thing to do would be to update the `TextField` component, otherwise we'd have to check the last character of `event.target.value` for `\n` and submit from there. kind of janky tbh

Reply:
@bethanyaconnor @dmcavoy do you have an opinion on removing this functionality that previously existed?

Reply:
What is the behavior we are deprecating?

Reply:
submitting the form whenever the user presses the enter key when focused in an input

Reply:
above pr has merged 🙌 

I'll add back the functionality and tests

Reply:
updated! I could only add the listener to the email and password fields. the radio button group does not have onKeyDown yet